lob/resource.py

```diff
@@ -170,6 +171,19 @@ def create(cls, **params):
         params.pop('objects', None)
         return super(Job, cls).create(**params)
 
+class Letter(ListableAPIResource, CreateableAPIResource):
+    endpoint = '/letters'
+    @classmethod
+    def create(cls, **params):
+        if isinstance(params, dict):
+            if 'from_address' in params:
+                params['from'] = params['from_address']
```

Comment:
is there  a reason we call the vars "to_address" and "from_address" and not "to" and "from"